unfccc.csl

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" demote-non-dropping-particle="sort-only" default-locale="en-GB" page-range-format="expanded">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>UNFCCC</title>
+    <title-short>UNFCCC</title-short>
+    <id>http://www.zotero.org/styles/unfccc</id>
+    <link rel="self" href="http://www.zotero.org/styles/unfccc"/>
+    <link href="https://www.zotero.org/styles/harvard-kings-college-london" rel="template"/>
```

Comment:
Thanks. I added the OneDrive link as a second documentation link and put in your name as author: https://github.com/citation-style-language/styles/commit/2d8887839f26e1198cb0b6b49799d15fe0613a43